aim/db/migration/alembic_migrations/versions/3ef7607a41b0_contracts.py

```diff
@@ -42,7 +42,9 @@ def upgrade():
         sa.PrimaryKeyConstraint('aim_id'),
         sa.UniqueConstraint('tenant_name', 'name',
                             name='uniq_aim_filters_identity'),
-        sa.Index('idx_aim_filters_identity', 'tenant_name', 'name'))
+        sa.Index('idx_aim_filters_identity', 'tenant_name', 'name'),
+        sa.ForeignKeyConstraint(
+            ['tenant_name'], ['aim_tenants.name'], name='fk_tn'))
```

Comment:
I think the FK constraint names have to be unique in MySQL. Can you confirm that using the same name for all constraints is OK?